app/views/works/show.html.erb

```diff
@@ -6,8 +6,9 @@
   <%= render Elements::SpinnerComponent.new %>
 <% end %>
 
-<%= render Elements::Tables::TableComponent.new(id: 'title-table', classes: 'table-show mb-5', label: 'Title') do |component| %>
+<%= render Elements::Tables::TableComponent.new(id: 'title-table', classes: 'table-work mb-5', label: 'Title and contact') do |component| %>
```

Comment:
I don't think the table-show class should have changed.

Reply:
I don't think that was deliberate, maybe part of rebasing. I'll fix. Thanks for catching this.